packages/terra-status-view/src/StatusView.module.scss

```diff
@@ -88,18 +107,15 @@
     padding-right: 0.357rem;
     padding-top: var(--terra-status-view-actions-padding-top, 1.429rem);
     width: 100%;
+  }
 
-    @media screen and (min-width: 768px) {
-      flex-direction: row;
-    }
-
-    > button {
-      flex: var(--terra-status-view-actions-button-flex, 0 1 auto);
-      flex-shrink: 1;
-      margin-left: var(--terra-status-view-actions-button-margin-left, 0.357rem);
-      margin-right: var(--terra-status-view-actions-button-margin-right, 0.357rem);
-      margin-top: var(--terra-status-view-actions-button-margin-top, 0);
-      max-width: 100%;
-    }
+  .button {
+    flex: var(--terra-status-view-actions-button-flex, 0 1 auto);
+    flex-shrink: 1;
+    margin-bottom: var(--terra-status-view-actions-button-margin-bottom, 0.71428rem);
+    margin-left: var(--terra-status-view-actions-button-margin-left, 0.357rem);
+    margin-right: var(--terra-status-view-actions-button-margin-right, 0.357rem);
+    margin-top: var(--terra-status-view-actions-button-margin-top, 0);
```

Comment:
oops - made a mistake. don't need a new theming variable. subtract back https://github.com/cerner/terra-core/pull/2453/commits/20e999e6ff16ee9dc907dc3470a6615482c10492 to remove margin-bottom again.
Then just change `.button { margin-top:` to `0.71428rem` and reduce `.actions { padding-top:` to `0.71428rem` [(line 108)](https://github.com/cerner/terra-core/pull/2453/files#diff-b68e8f9acd496ddbab9eb7f3855aa53eR108) to have the same effect without adding anything new.

Reply:
https://github.com/cerner/terra-core/pull/2453/commits/807b3881725c3e0cc556ab6f14aa9c057260dce4